kolibri/plugins/coach/assets/src/routes/planExamRoutes.js

```diff
@@ -110,8 +110,8 @@ export default [
     name: PageNames.EXAM_CREATION_PREVIEW,
     path: '/:classId/plan/quizzes/new/preview/',
     component: PlanQuizPreviewPage,
-    handler: toRoute => {
-      showExamCreationPreviewPage(store, toRoute.params);
+    handler: (toRoute, fromRoute) => {
+      showExamCreationPreviewPage(store, toRoute.params, fromRoute);
```

Comment:
I note that `query` is never passed to the handler here? Is is passed somewhere else?

Reply:
In [the function definition ](https://github.com/learningequality/kolibri/blob/12fb9f65edc005cae68e8645883f8324355dbe12/kolibri/plugins/coach/assets/src/modules/examCreation/handlers.js#L182), `query = {}` is passed as the last param. I'm honestly not sure if that is sufficient, but that's how it has been, so perhaps it is? 🤷‍♀️